tests/users_tests/test_users_views.py

```diff
@@ -71,3 +71,13 @@ def test_authorized_user_pages_access(self, client, url, template_file, new_user
         response = client.get(f'/{url}/')
         assert response.status_code == 200
         assertTemplateUsed(response, template_file)
+
+    def test_access_to_valid_user_profile(self, client):
+        response = client.get('/users/OmerCohenShor/')
+        assert response.status_code == 200
+        assertTemplateUsed(response, 'users/profile_details.html')
+
+    @pytest.mark.parametrize('invalid_profile', ['GIBRISH_PROFILE_PRO', '', '12', '1', '-1', ''])
```

Comment:
Maybe you could give it a different name than `GIBRISH_PROFILE_PRO`?

Reply:
Done!